CMakeLists.txt

```diff
@@ -16,6 +16,16 @@
 
 cmake_minimum_required(VERSION 2.8.11)
 
+#
+# Force static C-Runtime (MSVC) / also for OpenSSL
+# See: https://cmake.org/Wiki/CMake_FAQ#How_can_I_build_my_MSVC_application_with_a_static_runtime.3F
+#
+set(CMAKE_USER_MAKE_RULES_OVERRIDE ${CMAKE_CURRENT_SOURCE_DIR}/c_flag_overrides.cmake)
```

Comment:
Not that I know, OpenSSl.dll would get its CRT loaded, and ualds.exe its CRT dll, if they are the same, great, then they are ref counted, otherwise you would have 2 crt loaded into your process.  That makes dlls pretty nice, aside from avoiding bloat.  At least I never had a problem taking a dependency on a dll build against msvcrt_90...